app-frontend/src/app/components/projects/annotateSidebarItem/annotateSidebarItem.html

```diff
@@ -50,21 +50,24 @@
 <div class="list-group-item" ng-if="!$ctrl.annotation.geometry || ($ctrl.editId === $ctrl.annotation.properties.id)">
   <div class='annotation-new'>
     <form>
-    <angucomplete-alt pause="100"
-                      initial-value="$ctrl.annotation.properties.label"
-                      selected-object="$ctrl.labelObj"
-                      placeholder="{{$ctrl.annotation.properties.label || 'Label name...'}}"
-                      local-data="$ctrl.labelInputs"
-                      search-fields="name"
-                      title-field="name"
-                      minlength="1"
-                      input-class="form-control annotation-label"
-                      match-class="annotation-highlight"
-                      override-suggestions="true"
-                      ng-class="{'is-invalid': $ctrl.isInvalid }">
-    </angucomplete-alt>
-    <textarea class="form-control"
+      <input type="text"
```

Comment:
We should hide the suggestions on `blur`.

Reply:
Turns out this will not work as we had hoped. When clicking on a suggestion, the `ng-blur` is triggered and hides the suggestions before the suggestions `ng-click` is evaluated. This results is the suggestion not being used. `ng-blur` will have to call a function the leverages `$timeout` with some delay to allow the click to register before the `div` is hidden.